media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit user-info vdr-plugin-2
+
+DESCRIPTION="VDR plugin: Show rdf Newsticker on TV"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI="http://vdr.websitec.de/download/${PN}/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+
+BDEPEND="acct-user/vdr"
+DEPEND="${BDEPEND}
+	>=media-video/vdr-1.2.6
+	net-misc/wget"
```

Comment:
I don't see wget used during emerge? Based on the *.c file, I guess, it only used in runtime?

Reply:
ditto